src/vmm/src/logger/metrics.rs

```diff
@@ -1146,6 +1061,27 @@ impl Serialize for SerializeToUtcTimestampMs {
     }
 }
 
+#[derive(Default, Debug)]
+// By using the below structure in FirecrackerMetrics it is easy
+// to serialise Firecracker app_metrics as a signle json object which
+// otherwise would have required extra string manipulation to pack
+// net as part of the same json object as FirecrackerMetrics.
+pub struct NetMetricsSerializer {}
+impl NetMetricsSerializer {
+    pub const fn new() -> Self {
+        Self {}
+    }
+}
+
+impl Serialize for NetMetricsSerializer {
```

Comment:
sure, NetMetricsProxy sounds better.

Reply:
I used NetMetricsSerializeProxy so that the name suggests this is to be used only as a proxy to serialize and nothing else should be added to this in the future.